churn

```diff
@@ -1,6 +1,17 @@
 #!/usr/bin/env php
 <?php
-require_once 'vendor/autoload.php';
+
+if (is_file($autoload = __DIR__.'/vendor/autoload.php')) {
+    require($autoload);
```

Comment:
Should be `require_one` still.

Reply:
What's the point?

Reply:
@jakzal I think @GrahamCampbell's point is that `require_once()` is a bit safer since it will only include the file one time. Probably not an issue here but might as well be safe and use it.

Reply:
That's why I prefer when people explain WHY they request changes. IMO in this case it doesn't matter if we use one or the other as this file is an entry point for the application and is only ever loaded once.

Reply:
Similarly, originally you did not explain why you changed it from what it was.